app/views/categories/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= render '/shared/user_navbar'%>
+<main class="container mx-auto px-5 mt-5  w-full">
+  <%= form_with(model: @category, url: categories_path, method: :post) do |f|%>
+    <div class= 'grid grid-col gap-3 w-full'>
+      <div class="field  w-full ">
+        <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: 'Name', class: 'border-b border-b-[#d1d1d1] p-3 focus:outline-none w-full' %>
+      </div>
+      <div class="field  w-full">
+        <%= f.text_field :icon, autofocus: true, autocomplete: "name", placeholder: 'Add your icon url here', class: 'border-b border-b-[#d1d1d1] p-3 focus:outline-none w-full' %>
+      </div>
+      <%= f.submit "Add Category",class:"btn btn-primary" %>
+    </div>
```

Comment:
# Congratulations, you have done well
![Capture](https://github.com/shafiuyushawu/budget-app/assets/63972670/25488836-7ebc-4c23-97c9-8e347a42edd3)

Reply:
Hi @OLIPLICHE,
Appreciates you.